flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/PulsarSourceOptions.java

```diff
@@ -204,6 +205,19 @@ private PulsarSourceOptions() {
                                             "In this case, a single consumer will still receive all the keys, but they may be coming in different orders.")
                                     .build());
 
+    public static final ConfigOption<Boolean> PULSAR_ENABLE_SOURCE_METRICS =
+            ConfigOptions.key(SOURCE_CONFIG_PREFIX + "enableMetrics")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "We won't expose the metrics from Pulsar Consumer if you disable this option.")
```

Comment:
```suggestion
                                            "The metrics from Pulsar Consumer are only exposed if you enable this option.")
```

Use positive statements instead of double negative.